README.md

```diff
@@ -357,7 +386,7 @@ This can be useful alongside `event='none'` when you only want to toolip to show
 
 ```hbs
 {{!--Binds the tooltip visibility to the showTooltip property--}}
-{{tooltip-on-component isShown=showTooltip}}
```

Comment:
The `showTooltip` property seemed like an unnecessary complication in this example when we are just trying to convey the fact that you can pass in `isShown` as a boolean.